docs/modules/ROOT/pages/usage.adoc

```diff
@@ -307,7 +307,28 @@ spec:
 
 ==== LDAP
 
-Currently not supported.
+Druid supports xref:home:concepts:authentication.adoc[authentication] of users against an LDAP server.
+This requires setting up an xref:home:concepts:authentication.adoc#authenticationclass[AuthenticationClass] for the LDAP server.
+
+NOTE: You can follow the xref:home:tutorials:authentication_with_openldap.adoc[] tutorial to learn how to create an AuthenticationClass for an LDAP server.
+
+Here is a reduced, non-functional YAML example, which shows the most essential parts which are needed to configure LDAP in the druid-operator:
+
+[source,yaml]
+----
+include::example$code/druid-ldap-authentication.yaml[]
+----
```

Comment:
```suggestion
Reference the AuthenticationClass in your DruidCluster resource:

[source,yaml]
----
include::example$code/druid-ldap-authentication.yaml[tag=druid]
----
```

Reply:
See [Trino](https://github.com/stackabletech/trino-operator/blob/main/docs/modules/usage_guide/pages/security.adoc#ldap) for reference